packages/desktop-client/src/components/manager/subscribe/common.js

```diff
@@ -39,9 +42,14 @@ export function useBootstrapped() {
         // A server hasn't been specified yet
         history.push('/config-server');
       } else {
-        let { error, bootstrapped } = await send('subscribe-needs-bootstrap');
+        let { error, bootstrapped, passwordDisabled } = await send('subscribe-needs-bootstrap');
         if (error) {
           history.push('/error', { error });
+        } else if (passwordDisabled) {
+          let { error } = await send('subscribe-sign-in', {});
+          if (!error) {
+            dispatch(loggedIn(true));
```

Comment:
What's `true` for? `loggedIn` doesn't take an argument

Reply:
This is a rememant of another implementation I tried. I've since removed the argument but I forgot to remove it from there as well.